CHANGELOG.md

```diff
@@ -237,4 +237,4 @@
 [@braindeaf]: https://github.com/braindeaf
 [@ckhall]: https://github.com/ckhall
 [@anamba]: https://github.com/anamba
```

Comment:
Can you please also add a reference to yourself here (that's what the link at the top will point to!)

Reply:
Oh yeah my bad. I din't notice this